src/modules/navigator/rtl_direct.cpp

```diff
@@ -279,7 +279,7 @@ void RtlDirect::set_rtl_item()
 			DestinationPosition dest{_destination};
 			dest.alt = loiter_altitude;
 
-			setMoveToPositionMissionItem(_mission_item, dest, rtl_heading_mode);
+			setLandMissionItem(_mission_item, dest, rtl_heading_mode);
```

Comment:
This is required such that the type goes to CMD_LAND? Do you see any other side effects from the change? E.g. the heading setpoint could now be different to what it was set previously in setMoveToPositionMissionItem() no?